types/repo/AnonymousMessageRepo/MessageAnonymouslyData.ts

```diff
@@ -4,6 +4,10 @@ import {Member} from './AnonymousChannelsData';
 export interface MessageAnonymouslyData {
   channel: ChannelData;
   members: Member[];
+  anon_user_info_emoji_name: string | null | undefined;
+  anon_user_info_emoji_code: string | null | undefined;
+  anon_user_info_color_name: string | null | undefined;
+  anon_user_info_color_code: string | null | undefined;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `anon_user_info_emoji_name`, `anon_user_info_emoji_code`, `anon_user_info_color_name`, and `anon_user_info_color_code` are nullable and can be undefined. This could potentially lead to runtime errors if not properly checked before usage. Make sure to add null and undefined checks wherever these fields are used.
```diff
-  anon_user_info_emoji_name: string | null | undefined;
-  anon_user_info_emoji_code: string | null | undefined;
-  anon_user_info_color_name: string | null | undefined;
-  anon_user_info_color_code: string | null | undefined;
+  anon_user_info_emoji_name?: string;
+  anon_user_info_emoji_code?: string;
+  anon_user_info_color_name?: string;
+  anon_user_info_color_code?: string;
```
This change makes the fields optional instead of allowing them to be null or undefined, which is a more idiomatic way to handle this in TypeScript.


<!-- This is an auto-generated comment by OSS CodeRabbit -->